### PR TITLE
add comment

### PR DESCRIPTION
**Pull request by mkoeppe**

### CODE REVIEW

File: QNormalizModule.cpp [hunk 1]
```diff
@@ -547,7 +547,12 @@ static PyObject* _NmzQCone_internal(PyObject * args, PyObject* kwargs)
 
     map <InputType, vector< vector<NumberFieldElem> > > input;
     
-    PyDict_DelItemString(kwargs,"number_field");
+    PyDict_DelItemString(kwargs,"number_field");          // Doesn't
```
Comment (sebasguts) [inline QNormalizModule.cpp:550]: What is this comment supposed to mean?

**Review by sebasguts — CHANGES_REQUESTED** [review]

File: QNormalizModule.cpp (hunk 1 above)
Comment (sebasguts) [reply QNormalizModule.cpp:550]: Do you mean that the `kwargs` are not owned by the function? I guess it is still save to manipulate this dict here. I see no reason why it is not.

Comment (mkoeppe) [reply QNormalizModule.cpp:550]: Sorry for the cryptic comment. I was on an airplane. Example:
{{{
>>> kwargs = { 'number_field': "min_poly (a^2 - 5) embedding [2.236067977499789 +/- 8.06e-16]", 'cone': [[[1],[0,1]],[[1],[-1]]] }
>>> import PyQNormaliz_cpp
>>> P = PyQNormaliz_cpp.NmzCone(**kwargs)
>>> Q = PyQNormaliz_cpp.NmzCone(**kwargs)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
QNormaliz.interface_error: no number field data given
}}}

Comment (sebasguts) [reply QNormalizModule.cpp:550]: Oh, that is bad. I was convinced the dict itself would have been copied. I will fix this!

**Comment by sebasguts** [comment]
I fixed this and released 1.3.

**Pull request closed without merging**